test/easy_localization_test.dart

```diff
@@ -520,4 +520,33 @@ void main() {
       });
     });
   });
+
+  group('onKeyNotFound', () {
+    var r = EasyLocalizationController(
+        forceLocale: const Locale('en'),
+        supportedLocales: const [Locale('en'), Locale('fb')],
+        fallbackLocale: const Locale('fb'),
+        path: 'path',
+        useOnlyLangCode: true,
+        useFallbackTranslations: true,
+        onLoadError: (FlutterError e) {
+          log(e.toString());
+        },
+        saveLocale: false,
+        assetLoader: const JsonAssetLoader());
+
+    String? translationMissingKey;
+    setUpAll(() async {
+      r.loadTranslations();
+      Localization.load(const Locale('en'), translations: r.translations,
+          onLocaleKeyNotFound: ((key, locale) {
+        translationMissingKey = key;
+      }));
+    });
+
+    test('should call onLocaleKeyNotFound', () {
```

Comment:
lets add another test to verify that the method is NOT called when the translation exists